wikipedia/challenges/default.json

```diff
@@ -16,25 +16,25 @@
       "operation": "check-cluster-health"
     },
     {
-      "name": "index-documents",
-      "operation": "index-documents",
-      "warmup-time-period": {{bulk_warmup | default(40) | int}},
-      "clients": {{bulk_indexing_clients | default(5)}}
+      "name": "initial-documents-indexing",
+      "operation": "initial-documents-indexing",
+      "warmup-time-period": {{initial_indexing_bulk_warmup | default(40) | int}},
+      "clients": {{initial_indexing_bulk_indexing_clients | default(5) | int}}
```

Comment:
See suggestion above for renaming to `initial_indexing_bulk_clients`, if applicable.